### PR TITLE
Input with type number should return number after value change

### PR DESCRIPTION
**Pull request by Madhu94**
Attempting to close https://github.com/plotly/dash/issues/124. @chriddyp, please let me know if I should do something else here.

### CODE REVIEW

File: src/components/Input.react.js [hunk 1]
```diff
@@ -21,15 +21,22 @@ export default class Input extends Component {
     render() {
         const {
             fireEvent,
-            setProps
+            setProps,
+            inputType
```
Comment (chriddyp) [inline src/components/Input.react.js]: I believe that this should be `type` to match https://github.com/plotly/dash-core-components/pull/100/files#diff-368d71637f90189b9dab46fb80b2c1f1R78

**Comment by chriddyp** [comment]
This looks great @Madhu94, thank you! Just one small comment about the `prop` name.

**Comment by Madhu94** [comment]
@chriddyp Is this good to go now ?

**Comment by chriddyp** [comment]
Looks great @Madhu94 , thank you!

**Pull request merged**